src/driver/hermes_cdev.c

```diff
@@ -363,6 +366,29 @@ static long hermes_download_program(struct hermes_env *env,
 	return 0;
 }
 
+static long hermes_get_retval(struct hermes_env *env,
+		struct hermes_prog_retval_ioctl_argp *argp)
```

Comment:
This isn't a good idea, it breaks support with io_uring.

We should only support return codes that are 0 or a number indicating a failure. If the failure happens we should map it into an appropriate errno and return that from fsync() or write().